<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_close">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>CloseEdit Signal/Slot F4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_styleSheetSlotButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit StyleSheet slot Button</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(138, 226, 52);
color: #fff</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_showNormal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowNormal connect Signal to Slot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_styleSheetMainWindow">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>StyleSheet MainWindow slot button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_showMaximized">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowMaximized Edit Signal/Slot F4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_changeTextLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Text Label 
connect SignalSlot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Texto inicial</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bt_close</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_showMaximized</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMaximized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
